Assets/Scripts/Core/Controllers/GridCombatSystem.cs

```diff
@@ -119,6 +119,9 @@ private enum MouseButtons {
 				GameEnded?.Invoke(this, false);
 			} else {
 				redTeamList_.Remove(unit);
+				if(redTeamList_.Count <= 0)	{

```

Comment:
Seems like an erroneous tab was inserted after the `)`, can you remove it?

Reply:
Strange vscode rendered it as an space but it had the unicode character for the tab there if you looked very closely. Should be fixed now.

Reply:
Yeah github rendered it funny and upon closer inspection it turned out to be a tab. It looks fixed now.